builder/builder.go

```diff
@@ -286,7 +286,7 @@ func (b *Builder) runBuildingJob(slotCtx context.Context, proposerPubkey boostTy
 	}
 
 	// resubmits block builder requests every second
-	runRetryLoop(ctx, 500*time.Millisecond, func() {
+	runRetryLoop(ctx, 1*time.Second, func() {
```

Comment:
This should not impact the relay rate limit, as it's a loop scheduling new blocks and not a loop controlling how often they are submitted
Again, would be nice to make it either dynamic based on load or configurable - changing from one hardcoded value to another won't bring much benefit